### PR TITLE
Added capabilities for Artic Nova 7 mirophone(tested and working) and…

### PR DESCRIPTION
**Pull request by nicola02nb**
… volume limiter(tested and working); pre inserted instruction for bluetooth capabilities(tested,not working)

### Changes made

Added features for `Steelseries Artic Nova 7`:
- microphone mute led brightness
- microphone volume
- volume limiter

Pre added instructions to `main.c`, `device.h` and `device.c` for possible future implementation of:
- bluetooth when powered on
- bluetooth call volume

### Checklist

- [ ] I adjusted the README (if needed)
- [x] For new features in HeadsetControl: I discussed it beforehand in Issues or Discussions and adhered to the [wiki](https://github.com/Sapd/HeadsetControl/wiki/Development#adding-a-new-feature-to-the-headsestcontrol-application-itself)

### CODE REVIEW

**Comment by Sapd** [comment]
I will do a `make format` before merging so dont worry about the code style.

The other commented out functions dont work at all?

File: src/device.c [hunk 1]
```diff
@@ -45,5 +51,8 @@ const char capabilities_str_short[NUM_CAPABILITIES]
           [CAP_EQUALIZER_PRESET]               = 'p',
           [CAP_EQUALIZER]                      = 'e',
           [CAP_MICROPHONE_MUTE_LED_BRIGHTNESS] = 't',
-          [CAP_MICROPHONE_VOLUME]              = 'o'
+          [CAP_MICROPHONE_VOLUME]              = 'o',
+          [CAP_VOLUME_LIMITER]                 = 'g'//,
+          //[CAP_BT_WHEN_POWERED_ON]             = 'h',
+          //[CAP_BT_CALL_VOLUME]                 = 'c'
```
Comment (Sapd) [inline src/device.c:57]: The `capabilities_str_short` is basically used for the deprecated short output future.

I think I will replace it with \0 for the new features, and add a check here https://github.com/Sapd/HeadsetControl/blob/f64ac48384cbb7df3878749cbe9635adefdae84c/src/output.c#L728 that it does not output something on \0

Comment (nicola02nb) [reply src/device.c:57]: > The `capabilities_str_short` is basically used for the deprecated short output future.
> 
> I think I will replace it with \0 for the new features, and add a check here
> 
> https://github.com/Sapd/HeadsetControl/blob/f64ac48384cbb7df3878749cbe9635adefdae84c/src/output.c#L728
> 
> that it does not output something on \0

There I've putted some random letters as a placeholder... if you want to remove them after my edits, you're free to do it.

**Comment by nicola02nb** [comment]
I also forgot to put the help section for the new command...

File: src/main.c [hunk 2]
```diff
@@ -449,6 +453,22 @@ void print_help(char* programname, struct device* device_found, bool _show_all)
     }
     // ------
 
+    // ------ Category: Bluetooth
+    bool show_bt_when_powered_on = show_all || has_capability(device_found->capabilities, CAP_BT_WHEN_POWERED_ON);
+    bool show_bt_call_volume     = show_all || has_capability(device_found->capabilities, CAP_BT_CALL_VOLUME);
+
+    if (show_rotate_to_mute || show_microphone_mute_led_brightness || show_microphone_volume) {
```
Comment (Sapd) [inline src/main.c:460]: Small mistake here: `show_rotate_to_mute` probably left over from copying the code

Comment (nicola02nb) [reply src/main.c:460]: Yessir, fixed

Comment (Sapd) [reply src/main.c:460]: Ahh its still there :)

Comment (nicola02nb) [reply src/main.c:460]: Damn didn't see that

Comment (nicola02nb) [reply src/main.c:460]: Sorry :(

**Comment by nicola02nb** [comment]
Can you explain me line 32 of the code style ceck?

**Comment by Sapd** [comment]
> Can you explain me line 32 of the code style ceck?

Yeah there is a whitespace at the end of the line

```
-    if (num == 8) <-- here
+    if (num == 8)
```

But does not really matter, I will also fix the enum from above later and then also re run a format over it

**Comment by nicola02nb** [comment]
> > Can you explain me line 32 of the code style ceck?
> 
> Yeah there is a whitespace at the end of the line
> 
> ```
> -    if (num == 8) <-- here
> +    if (num == 8)
> ```
> 
> But does not really matter, I will also fix the enum from above later and then also re run a format over it

Oh 😅

**Comment by nicola02nb** [comment]
Any info on first error?

**Comment by Sapd** [comment]
Looks fine to me. I think the whitespace check is quite simple and just does a diff to the commit before. It just sees that there is a diff and that its why it outputs it. It will probably go away when I merge it later.

**Comment by nicola02nb** [comment]
Ok, I hope i wasn't an issue with all this commits 😅

**Comment by Sapd** [comment]
Thank you! well done

**Pull request merged**